packages/web/src/components/swap/setting-menu-modal/SettingMenuModal.tsx

```diff
@@ -108,9 +108,9 @@ const SettingMenuModal: React.FC<SettingMenuModalProps> = ({
       if (value === "") {
         changeSlippage("0");
         setPrevios("0");
-      } else if (Number(value) > 30) {
-        changeSlippage("30");
-        setPrevios("30");
+      } else if (Number(value) > 50) {
```

Comment:
Why not define and use a value for maximum slippage?

Reply:
Yes, it would be nice to use constants to avoid hardcoding

Reply:
Gotcha! I fixed it.